test/OpenTelemetry.Exporter.Geneva.Tests/GenevaMetricExporterTests.cs

```diff
@@ -40,7 +41,7 @@ public void InvalidConnectionString(string connectionString)
         var exporterOptions = new GenevaMetricExporterOptions() { ConnectionString = connectionString };
         var exception = Assert.Throws<ArgumentException>(() =>
         {
-            using var exporter = new GenevaMetricExporter(exporterOptions);
+            using var exporter = new TlvMetricExporter(exporterOptions);
```

Comment:
These tests should use GenevaMetricExporter and not TlvMetricExporter as `GenevaMetricExporter` is the public class.